packages/suite/src/support/messages.ts

```diff
@@ -8806,6 +8826,10 @@ export default defineMessages({
         id: 'TR_STAKE_REWARDS',
         defaultMessage: 'Rewards',
     },
+    TR_STAKE_EXPECTED_REWARDS: {
+        id: 'TR_STAKE_EXPECTED_REWARDS',
+        defaultMessage: 'Expected Rewards per 1 epoch (~3 days)',
```

Comment:
```suggestion
        defaultMessage: 'Expected rewards per 1 epoch (~3 days)',
```
Should 3 days be hardcoded?